test/rtos_drivers/usb/src/individual_tests/usb/local/usb_sof_test.c

```diff
@@ -38,8 +38,9 @@ static void timeout_cb(TimerHandle_t xTimer)
     sof_timeout = true;
 }
 
-bool tud_xcore_sof_cb(uint8_t rhport)
+bool tud_xcore_sof_cb(uint8_t rhport, uint32_t cur_time)

```

Comment:
does it matter that the name of the variable is different between its weak declaration and its definition here?

Reply:
Actually looking at the other review, you've used "cur_time" as the variable name everywhere else - suggest changing the declaration to "cur_time" to harmonise throughout

Reply:
I'll change it. About your earlier comment I don't think the variable names differently in the weak declaration matters.